src/types/client/checker.d.ts

```diff
@@ -0,0 +1,59 @@
+export interface Config {
+  id: number
```

Comment:
Let's make this a string, letting the user specify the id. We then check in the backend whether the id is unique.

It will also be good if this interface is available to the server to, so perhaps define it at `src/types` then alias it in the client? I can then do likewise server end.

Reply:
@LoneRifle sorry, how do I alias it in client?

Reply:
maybe...
```js
export { Config } from '..'
```

Reply:
or alternatively, just refer to the type directly from `components/Checker`